ros/bouncy/ubuntu/bionic/ros-core/Dockerfile

```diff
@@ -15,10 +15,10 @@ RUN apt-get update && apt-get install -q -y \
     && rm -rf /var/lib/apt/lists/*
 
 # setup keys
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 4B63CF8FDE49746E98FA01DDAD19BAB3CBF125EA
 
 # setup sources.list
-RUN echo "deb http://packages.ros.org/ros2/ubuntu bionic main" > /etc/apt/sources.list.d/ros2-latest.list
+RUN echo "deb http://snapshots.ros.org/bouncy/final/ubuntu bionic main" > /etc/apt/sources.list.d/ros2-snapshots.list
```

Comment:
Wasn't this change already PR'ed to the library?
Otherwise, we'll need to stage/separate this changes from comment-outs in the manifest.

Reply:
Only for distros that were listed as EOL at the time of the PR. So Bouncy was not included in the previous upstream PR.

> Otherwise, we'll need to stage/separate this changes from comment-outs in the manifest.

I was thinking of submitting a single PR removing the rebuilt EOLs and at the same time switching Bouncy to snapshots repo.
Once Bouncy rebuilt, open an other PR to remove it from the manifest

Reply:
Sounds good then